test/streams_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative "helper"
+
+class TestStreams < Minitest::Test
+  include Helper::Client
+
+  def test_read_a_trimmed_entry
+    r.xgroup(:create, 'k1', 'g1', '0', mkstream: true)
+    entry_id = r.xadd('k1', { value: 'v1' })
+
+    assert_equal r.xreadgroup('g1', 'c1', 'k1', '>'), { 'k1' => [[entry_id, { 'value' => 'v1' }]] }
```

Comment:
The order of arguments is `assert_equal expectation, actual_value`.

Reply:
I fixed it.
FYI, I based my suite on `ssl_test` that also has this problem 😅
https://github.com/redis/redis-rb/blob/6542934f01b9c390ee450bd372209a04bc3a239b/test/ssl_test.rb#L19